bcda/client/bluebutton.go

```diff
@@ -379,7 +380,7 @@ func (bbc *BlueButtonClient) getURL(path string, params url.Values) (*url.URL, e
 	return u, nil
 }
 
-func addDefaultRequestHeaders(req *http.Request, reqID uuid.UUID, jobID, cmsID string) {
+func addDefaultRequestHeaders(req *http.Request, reqID uuid.UUID, jobData models.JobEnqueueArgs) {
```

Comment:
since we are returning 3 values from the JobEnqueueArgs model, I removed the individual parameters and opted to pass in the whole type instead.